tests/test_path_analysis.py

```diff
@@ -0,0 +1,153 @@
+import os
+import os.path
+import numpy as np
+import networkx as nx
+import pytest
+from numpy.testing import assert_almost_equal, assert_equal
+from pyinteraph import path_analysis as pa
+
+# Load files
+@pytest.fixture
+def ref_dir(request):
+    return os.path.join(request.fspath.dirname, '../examples')
+
+@pytest.fixture
+def data_files(ref_dir):
+    return { 
+             'pdb' : os.path.join(ref_dir, 'sim.prot.twochains.pdb'),
+             'psn' : os.path.join(ref_dir, 'sc-graph_twochains_all.dat')
+           }
+
+@pytest.fixture
+def data(data_files):
+    return pa.build_graph(data_files['psn'], data_files['pdb'])
+
+@pytest.fixture
+def source(data):
+    return pa.convert_input_to_list(user_input = "A1:A2,A57",
+                                    identifiers = data[0])
+
+@pytest.fixture
+def target(data):
+    return pa.convert_input_to_list(user_input = "B1042",
+                                    identifiers = data[0])
+
```

Comment:
Leave only one blank line between the test functions (or be consistent in leaving two).